src/rust/engine/src/cffi_build.rs

```diff
@@ -23,6 +23,33 @@ use std::io::{Read, Result};
 use std::path::{Path, PathBuf};
 
 fn main() {
+  // We depend on grpcio, which uses C++.
+  // On Linux, with g++, some part of that compilation depends on
+  // __gxx_personality_v0 which is present in the C++ standard library.
+  // I don't know why. It shouldn't, and before grpcio 0.2.0, it didn't.
+  //
+  // So we need to link against the C++ standard library. Nothing under us
+  // in the dependency tree appears to export this fact.
+  // Ideally, we would be linking dynamically, because statically linking
+  // against libstdc++ is kind of scary. But we're only doing it to pull in a
+  // bogus symbol anyway, so what's the worst that can happen?
+  //
```

Comment:
Thanks a ton for diving into this.

Maybe we can link this to a ticket to spike switching to `tower-grpc` in a few cycles?